source/net/translation.ooc

```diff
@@ -10,63 +10,66 @@ import berkeley, Socket, Address
 
 Inet: class {
 	ntop: static func (addressFamily: Int, address: Pointer, destination: CString, destinationSize: UInt) -> CString {
-		version (!windows) {
+		result: CString
+		//version (!windows) {
 			// use built-in!
-			return inet_ntop(addressFamily, address, destination, destinationSize)
-		}
+			// TODO: Using the built-in inet_ntop results in a gcc warning because inet_ntop returns a const char*,
+			// and the const qualifier is not supported by rock...
+			//result = inet_ntop(addressFamily, address, destination, destinationSize)
+		//} else {
+			// Roll our own implementation, based on:
+			// https://github.com/pkulchenko/luasocket/blob/5a58786a39bbef7ed4805821cc921f1d40f12068/src/inet.c#L512
```

Comment:
As the comment says, `inet_ntop` returns `const char*`, but the closest thing we have is `CString` which is a cover of `char*`...